Examples/CodePushDemoApp-pre0.49/package.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "CodePushDemoApp",
+	"version": "0.0.1",
+	"private": true,
+	"scripts": {
+		"start": "react-native start",
+		"test": "jest"
+	},
+	"dependencies": {
+		"react": "16.0.0-alpha.12",
+		"react-native": "^0.47.1",
+		"react-native-code-push": "file:../../",
```

Comment:
RNCP version should be fixed here

Reply:
This is done in accordance with previous implementation (seems make sense to test against local copy of RN)

Reply:
@max-mironov okay i got it